code/game/g_roff.cpp

```diff
@@ -238,34 +277,176 @@ static void G_RoffNotetrackCallback( gentity_t *cent, const char *notetrack)
 			useOrigin[1] += up[1]*parsedOffset[2];
 			useOrigin[2] += up[2]*parsedOffset[2];
 
+#if !NDEBUG
```

Comment:
Debug prints no longer needed? Suggest removing them all

Reply:
@Razish ...would you not want to retain those for debug builds?

Reply:
@Razish -- ok... the code has been updated to remove the debug-build prints and replaced with an if statement using g_developer->integer.  Should be good to go now.